File: EmployerTerminal/Models/Image.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EmployerTerminal.Models
{
    public class Image
    {
        public int Image_id { get; set; }
        public int Project_id { get; set; }
        public string Image_url { get; set; }
        public string Image_alt_text { get; set; }
    }
}

File: employerterminal.server/models/Image.js
const mongoose = require('mongoose');

const imageSchema = mongoose.Schema({
  image_id: { type: Number, required: true },
  project_id: { type: Number, ref: 'Project', required: true },
  image_url: { type: String, required: true },
  image_alt_text: { type: String, required: true }
},
  {collection: 'image_table'}
);

module.exports = mongoose.model('Image', imageSchema);